Makefile

```diff
@@ -72,7 +72,7 @@ CONTAINER_ARCH_TARGETS=$(addprefix container-,$(ALL_ARCHITECTURES))
 container:
 	# Pull base image explicitly. Keep in sync with Dockerfile, otherwise
 	# GCB builds will start failing.
-	docker pull golang:1.19.8
+	docker pull golang:1.20.5
```

Comment:
Can you please remove this unrelated change?

Reply:
Done. :)